share/p2p/shrexnd/pb/share.proto

```diff
@@ -28,5 +28,6 @@ message Row {
 message Proof {
   int64 start = 1;
   int64 end = 2;
-  repeated bytes Nodes = 3;
+  repeated bytes nodes = 3;
+  bytes hashleaf = 4;
```

Comment:
Optional?

Reply:
I would say it is optional to make it optional 😆 . There is no check for hashleaf being nil or zero-value on client side, since proof construction is based on status code model